CMakeLists.txt

```diff
@@ -471,9 +471,6 @@ if(GCC OR CLANG)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${C_CXX_FLAGS} -Wmissing-declarations")
 
   if(NOT MSVC)
-    if (NOT CMAKE_CXX_STANDARD)
-      set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
-    endif ()
```

Comment:
`CMAKE_CXX_STANDARD` is being set above [around line 100](https://github.com/aws/aws-lc/blob/main/CMakeLists.txt#L97).